plugins/home/src/homePageComponents/FeaturedDocs/FeaturedDocs.tsx

```diff
@@ -0,0 +1,168 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import {
+  LinkButton,
+  EmptyState,
+  Link,
+  InfoCard,
+  Progress,
+  ErrorPanel,
+} from '@backstage/core-components';
+import { catalogApiRef, CatalogApi } from '@backstage/plugin-catalog-react';
+import { useApi } from '@backstage/core-plugin-api';
+import { EntityFilterQuery } from '@backstage/catalog-client';
+import { ClassNameMap } from '@material-ui/styles';
+
+import { makeStyles, Theme, Typography } from '@material-ui/core';
+
+/**
+ * Props customizing the <FeaturedDocs/> component.
+ *
+ * @public
+ */
+export type FeaturedDocsProps = {
+  /** The entity filter used to display only the intended item/s */
+  filter: EntityFilterQuery;
+  /** An optional color which can be customized through themes */
+  color?: 'inherit' | 'primary' | 'secondary';
+  /** An optional ClassNameMap created with makeStyles */
+  customStyles?: ClassNameMap<string>;
+  /** An optional ReactNode for empty states */
+  emptyState?: React.ReactNode;
+  /** An optional path to set for entity entry  */
+  path?: string;
+  /** An optional limit to set for link destination  */
+  responseLimit?: number;
+  /** An optional string to customize sublink text */
+  subLinkText?: string;
+  /** An optional string or ReactNode to customize the card title */
+  title?: React.ReactNode | string;
+};
+
+const useStyles = makeStyles<Theme>(
+  theme => ({
+    docDescription: {
+      fontSize: theme.typography.body1.fontSize,
+      fontWeight: theme.typography.body1.fontWeight,
+      marginBottom: theme.spacing(2),
+      marginTop: theme.spacing(2),
+    },
+    docSubLink: {
+      fontSize: theme.typography.subtitle1.fontSize,
+      fontWeight: theme.typography.subtitle1.fontWeight,
+      lineHeight: theme.typography.subtitle1.lineHeight,
+    },
+    docsTitleLink: {
+      fontSize: theme.typography.h6.fontSize,
+      fontWeight: theme.typography.h6.fontWeight,
+      lineHeight: theme.typography.h6.lineHeight,
+    },
+  }),
+  { name: 'BackstageFeaturedDocs' },
+);
+
+/**
+ * A component to display specific Featured Docs.
+ *
+ * @public
+ */
+export const FeaturedDocs = (props: FeaturedDocsProps) => {
+  const {
+    color,
+    customStyles,
+    emptyState,
+    filter,
+    path,
+    responseLimit,
+    subLinkText,
+    title,
+  } = props;
+  const linkText = subLinkText || 'LEARN MORE';
+  const defaultStyles = useStyles();
+  const styles = customStyles || defaultStyles;
+  const catalogApi: CatalogApi = useApi(catalogApiRef);
+  const {
+    value: entities,
+    loading,
+    error,
+  } = useAsync(async () => {
+    const response = await catalogApi.getEntities({
```

Comment:
small suggestion which consumes less resources backend-side 😉

```suggestion
    const response = await catalogApi.queryEntities({
```

Reply:
@vinzscam, thank you for the suggestion! I did not know about `queryEntities`, but one thing I did notice from the [API description](https://github.com/backstage/backstage/blob/513bb37fa0033ef7c7a1ac46c04be217cde0bc1d/packages/catalog-client/src/types/api.ts#L522) is that this is set up for paginated responses.
I would have to make some logic updates to handle that case for pagination, but this card is only meant to view  1-5 documents, past that the UI should turn to some sort of [carousel](https://react-bootstrap.netlify.app/docs/components/carousel/) or paginator. 
The current limit for `getEntities` is defaulted to 10 -- is it ok if we leave `getEntities`? 

Either way, I can think of a few places to use this in our implementation at Autodesk 😄 - so thank you!

Reply:
I think `queryEntities` should work out of the box, you can ignore all the pagination payload in your case, as you want to retrieve maximum `limit` resources.
But you are right! Since you are passing a fixed limit it is totally fine to use `getEntities` 😄